.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - SFDX_AUTOUPDATE_DISABLE=true
     - SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
     - SFDX_DOMAIN_RETRY=300
+    - CODE_TESTS_WORKSPACE=$TRAVIS_BUILD_DIR/packages/system-tests/sfdx-simple
```

Comment:
Assuming you mean the system-tests, they shouldn't fail because they don't use the vscode test module. `npm test` in all the modules except in system-tests run `node ./node_modules/vscode/bin/test`. System-tests runs `node out/src/main.js` which will eventually run into the Application constructor that lets you pass in the workspace you want to use.